docs/api-docs/building-checkouts/checkout-consignment.md

```diff
@@ -148,7 +148,7 @@ There are two distinct kinds of consignment updates. The first selects a fulfill
 
 <!-- theme: warning -->
 
-> You must choose: because changing the shipping address and weight can change available fulfillment options, you can't do both in the same call to the [Update a consignment endpoint](/api-reference/storefront/checkouts/checkout-consignments/checkoutsconsignmentsbycheckoutidandconsignmentidput).
+> You must choose the type of consignment update because changing the shipping address and weight can change available fulfillment options. You can't do both in the same call to the [Update a consignment endpoint](/api-reference/storefront/checkouts/checkout-consignments/checkoutsconsignmentsbycheckoutidandconsignmentidput).
```

Comment:
How about `one type` of consignment update? -- to contextualize the choice there as one of the `two distinct kinds` above.